src/qt/txhistorydialog.cpp

```diff
@@ -324,6 +326,13 @@ int TXHistoryDialog::PopulateHistoryMap()
         if (!IsMyAddress(mp_obj.getSender())) htxo.address = mp_obj.getReceiver();
         if (htxo.fundsMoved && IsMyAddress(mp_obj.getSender())) displayAmount = "-" + displayAmount;
 
+        if (type == EXODUS_TYPE_SIMPLE_SPEND) {
+            htxo.address = "Spend";
+            if (htxo.fundsMoved && !IsMyAddress(mp_obj.getReceiver())) {
+                displayAmount = "-" + displayAmount;
+            }
+        }
```

Comment:
Better to merge this with the above statements.

Reply:
The problem is displayAmount is define and initialize at line 320